libs/features/seotech/seotech.js

```diff
@@ -10,11 +16,24 @@ export function logError(msg) {
   });
 }
 
+export function parseVideoUrl(url, providers = VIDEO_OBJECT_PROVIDERS) {
```

Comment:
Is the parameter used anywhere?

Reply:
Yes, function uses both `url` and `providers`.